README.md

```diff
@@ -275,4 +275,4 @@ You can learn more about the Rollup API and configuration [here](https://github.
 ## Terms And Conditions
 
 - Any changes to this project must be reviewed and approved by the repository owner. For more information about contributing, see the [Contribution Guide](https://github.com/moltin/gatsby-demo-store/blob/master/.github/CONTRIBUTING.md).
-- For more information about the license, see [MIT License](https://github.com/moltin/js-sdk/blob/master/LICENSE).
+- For more information about the license, see [MIT License](https://github.com/moltin/js-sdk/blob/main/LICENSE).
```

Comment:
Replace "moltin" with "elasticpath"

Reply:
This will be done in a PR after transferring ownership.

Reply:
Thanks for explaining. Resolving this discussion.